tools/netboxr/netboxr_xml.xml

```diff
@@ -1,35 +1,46 @@
-<tool id="netboxr" name="netboxr" version="1.6.0">
-    <description>netboxr enables automated discovery of biological process modules by network analysis.</description>
+<tool id="netboxr" name="netboxr" version="1.6.0" profile="21.05">
```

Comment:
@bgruening The addition of "profile="21.05"" introduces some kind of bug. Do you know what the reason might be? Can the PR be merged without this?

Reply:
I think it was the wrong spelling of geneList

Reply:
I already changed that independently. There were also additional problems that I fixed, I'm just having issues committing the changes.

Reply:
Mh, sorry, just wanted to help.

You need to pull the latest changes before you can commit your changes.

The profile is important as it ensures stricter isolation of tasks and more security.